.github/workflows/documentation.yml

```
@@ -0,0 +1,213 @@
+      - name: Check for broken links
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with:
+          use-quiet-mode: 'yes'
+          use-verbose-mode: 'yes'
+          config-file: '.github/workflows/link-check-config.json'
+          folder-path: 'ai-docs/, specs/, interactive-bduf-orchestrator/docs/'
+
+      - name: Validate implementation prompt structure
+        run: |
+          echo "🔍 Validating implementation prompt structure..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)